### ISSUE TITLE
Model V2: Refine model

### ISSUE THREAD

**Issue by d-sbrgr** [d-sbrgr/DSPRO2-jassAIr#17]
Performance on first version was very good on validation set (99+%), but inference on real-life-images was quite bad.

Find causes for this behavior, improve training dataset and retrain model to yield higher accuracy.

- [ ] Organize data.yaml files

- [ ] Freeze backbone and fine-tune on real-world data (real_world_yolo)
- [ ] Train current model on current synthetic data -> early stop at 70% -> check on synthetic and real test data
- [ ] Document in report!

If no significant improvement:
- [ ] Train on new synthetic dataset with 2 classes (eichel 6, schilte konig) and green background (train: 200)
- [ ] Make sure model can set bounding boxes!
